docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -102,10 +102,11 @@ Prior to pushing your changes to Github, you can view a local build of the docum
 When you are ready for other people to review your work, perform the following tasks:
 
 1. Add your changes, which prepares your content for the next commit.
-1. Commit your changes.
-1. Push your changes to Github.
-1. Create a PR in Github.
-1. Add the `type/docs` label.
+2. Commit your changes.
+3. Push your changes to Github.
+4. Create a PR in Github.
+5. When writing the description for your PR, use [GitHub keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) for example, "Fixes issue #1234", to link your pull request to the issue and take advantage of GitHub automation for status updates and closing resolved issues.
+6. Add the `type/docs` label.  The docs squad uses this label to track documentation work.
```

Comment:
```suggestion
1. Commit your changes.
1. Push your changes to Github.
1. Create a PR in Github.
1. When writing the description for your PR, use [GitHub keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue), for example "Fixes #1234", to link your pull request to the issue and take advantage of GitHub automation for status updates and closing resolved issues.
1. Add the `type/docs` label, so the Docs team can track the issue.
```
Rationale for change: `1.` auto-numbers the steps, so when they change, one does not have to reorder the numbered list.

Reply:
Yep, found the setting that was automatically renumbering my lists.  Thanks for pointing it out. I was following best practices, but my tool was changing the numbering, and I would have never known if you hadn't said something.